hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/PravegaOutputFormat.java

```diff
@@ -85,6 +87,10 @@ protected PravegaOutputFormat(ClientFactory externalClientFactory) {
 
     @Override
     public RecordWriter<String, V> getRecordWriter(TaskAttemptContext context) throws IOException, InterruptedException {
+        return getRecordWriter(context, null);
+    }
+
+    public RecordWriter<String, V> getRecordWriter(TaskAttemptContext context, String segmentRoutingKey) throws IOException, InterruptedException {
```

Comment:
`segmentRoutingKey` is confusing. Why is it referring to a segment?

Reply:
fixed